tsconfig.json

```diff
@@ -1,18 +1,10 @@
 {
+  "extends": "@fingerprintjs/tsconfig-dx-team/tsconfig.json",
   "compilerOptions": {
     "outDir": "dist/src",
     "module": "es6",
     "moduleResolution": "node",
-    "target": "es2020",
-    "resolveJsonModule": true,
-    "importHelpers": true,
-    "allowSyntheticDefaultImports": true,
-    "noErrorTruncation": true,
-    "strict": true,
-    "noUnusedLocals": true,
-    "noUnusedParameters": true,
-    "noImplicitReturns": true,
-    "exactOptionalPropertyTypes": true
+    "target": "es2020"
```

Comment:
Let's add to the readme usage example that we should specify some fields like `module`, `moduleResolution` and `target` depending of the project. May be we can even recommend what to use depending of a case.|
Also it will be greater to mention in the readme of using `"jsx": "react-jsx"` field for jsx based projects and how it works (at least the link to the documentation).

Reply:
Done!